### ISSUE TITLE
Document `skip` parameter (or merge `depend_on`)

### ISSUE THREAD

**Issue by kennknowles** [django-denorm/django-denorm#34]
#13 added a crucial piece to avoid common infinite loops, but these are not mentioned in the docs

**Comment by prokaktus** [comment]
Yes. Docs should be updated. Outdated header `django-denorm v0.1 documentation »`, years, management commands.
